x-pack/plugins/lens/public/types.ts

```diff
@@ -522,6 +522,9 @@ export interface OperationMetadata {
   // document and an aggregated metric which might be handy in some cases. Once we
   // introduce a raw document datasource, this should be considered here.
   isStaticValue?: boolean;
+  // There are some aggregations, such as the percentile ranks, that accept multi values
+  // and they need to be treated with a different way. This flag is used to identify these operations
+  isMultiValuesAggregation?: boolean;
```

Comment:
This was solved differently for percentile - instead of handling this within Lens, there's a "single percentile" metric agg: https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/data/common/search/aggs/metrics/single_percentile.ts#L25

It seems like it would make sense to align here and introduce the same for percentile rank. What do you think @ppisljar @stratoula ?

Reply:
I don't have a strong feeling. When I was creating this PR it felt right to use the same function as the agg based visualizations but I am happy to also create a singlPercentileRank function.

Reply:
Done! Much better that way!